massgen/frontend/displays/textual_widgets/content_sections.py

```
@@ -0,0 +1,1971 @@
+            self._open_workspace()
+
+    def _copy_to_clipboard(self) -> None:
+        """Copy final answer to system clipboard."""
+        import platform
+        import subprocess
+
+        full_content = self.get_content()
+        try:
+            system = platform.system()
+            if system == "Darwin":
+                process = subprocess.Popen(["pbcopy"], stdin=subprocess.PIPE)
```

Ruff (0.14.13)

Starting a process with a partial executable path

(S607)